apps/api/src/app/models/categories/categories.interface.ts

```diff
@@ -0,0 +1,36 @@
+import { Document } from "mongoose";
+
+export interface LeanCategoriesView {
```

Comment:
extend title and value from CategoryPayload, and maybe  you can all id use with underscore and all interfaces make more reusable, on this file